packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.service.js

```diff
@@ -127,55 +132,76 @@ export default class DedicatedServerInterfacesService {
 
   getTasks(serverName) {
     return this.$http
-      .get(`/dedicated/server/${serverName}/task?function=${INTERFACE_TASK}`)
-      .then(
-        (response) => response.data,
-        () => [],
-      )
-      .then((taskIds) => ({
-        promise: this.waitAllTasks(
-          serverName,
-          taskIds.map((taskId) => ({ taskId })),
-        ),
-      }));
+      .get(`/dedicated/server/${serverName}/task`, {
+        headers: {
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+          'X-Pagination-Size': 5000,
+          'X-Pagination-Filter': `function:in=${INTERFACE_TASK},${INTERFACE_GROUP_TASK},${INTERFACE_UNGROUP_TASK}`,
+        },
+      })
+      .then(({ data }) => data);
   }
 
-  disableInterfaces(serverName, interfaces) {
-    return this.$q
-      .all(
-        interfaces
-          .filter((i) => i.enabled === true)
-          .map(
-            (i) =>
-              this.VirtualInterface.v6().disable(
-                {
-                  serverName,
-                  uuid: i.id,
-                },
-                {},
-              ).$promise,
-          ),
-      )
-      .then((tasks) => this.waitAllTasks(serverName, tasks));
+  waitTasks(serverName) {
+    return this.getTasks(serverName).then((tasks) =>
+      this.waitAllTasks(serverName, tasks),
+    );
   }
 
   setPrivateAggregation(serverName, name, interfacesToGroup) {
-    return this.Ola.v6().group(
-      { serverName },
-      {
+    return this.$http
+      .post(`/dedicated/server/${serverName}/ola/aggregation`, {
         name,
         virtualNetworkInterfaces: map(interfacesToGroup, 'id'),
-      },
-    ).$promise;
+      })
+      .then((task) => this.waitForTask(serverName, task.data.taskId));
   }
 
   setDefaultInterfaces(serverName, interfaceToUngroup) {
-    return this.Ola.v6().ungroup(
-      { serverName },
-      {
+    return this.$http
+      .post(`/dedicated/server/${serverName}/ola/reset`, {
         virtualNetworkInterface: interfaceToUngroup.id,
+      })
+      .then((task) => this.waitForTask(serverName, task.data.taskId));
+  }
+
+  waitForTask(serverName, taskId) {
+    return this.Poller.poll(
+      `/dedicated/server/${serverName}/task/${taskId}`,
+      {},
+      {
+        namespace: 'dedicated.server.interfaces.ola.aggregation',
+        method: 'get',
+        successRule: {
+          status: 'done',
+        },
+        errorRule: (task) => {
+          return task.status === 'ovhError' || task.status === 'customerError';
+        },
       },
-    ).$promise;
+    ).then(
+      () => true,
+      (error) => {
+        if (error.status === 404) {
+          return true;
+        }
+        return Promise.reject(error);
+      },
+    );
```

Comment:
```suggestion
    ).then(() => true))
     .catch((error) => (error.status === 404 ? true : Promise.reject(error)));
```

Also, you should avoid using `Promise.reject(error)` as you are in AngularJS cycle